inc/sitemaps/class-sitemap-image-parser.php

```diff
@@ -425,6 +393,92 @@ protected function get_absolute_url( $src ) {
 	}
 
 	/**
+	 * Returns the attachments for a gallery.
+	 *
+	 * @param int   $id      The post id.
+	 * @param array $gallery The gallery config.
+	 *
+	 * @return array The selected attachments
+	 */
+	protected function get_gallery_attachments( $id, $gallery ) {
+
+		// When there are posts to include.
+		if ( ! empty( $gallery['include'] ) ) {
+			$_attachments = $this->get_attachments(
```

Comment:
I think the control flow here would me much clearer if everything inside this if was extracted to it's own function.